cli/commands/hclfmt/action_test.go

```diff
@@ -263,6 +263,41 @@ func TestHCLFmtFile(t *testing.T) {
 	}
 }
 
+func TestHCLFmtStdin(t *testing.T) {
+	t.Parallel()
+
+	realStdin := os.Stdin
+	realStdout := os.Stdout
+
+	tempStdoutFile, err := os.CreateTemp(t.TempDir(), "stdout.hcl")
+	defer func() {
+		_ = tempStdoutFile.Close()
+	}()
+	require.NoError(t, err)
+
+	os.Stdout = tempStdoutFile
+	defer func() { os.Stdout = realStdout }()
+
+	os.Stdin, err = os.Open("../../../test/fixture-hclfmt-stdin/terragrunt.hcl")
```

Comment:
Path format was updated to `test/fixtures/...`, I think the branch should be synchronized with main to get latest changes